src/GitLink.Tests/GlobalInitialization.cs

```diff
@@ -1,17 +1,16 @@
 ﻿// --------------------------------------------------------------------------------------------------------------------
 // <copyright file="GlobalInitialization.cs" company="CatenaLogic">
-//   Copyright (c) 2014 - 2014 CatenaLogic. All rights reserved.
+//   Copyright (c) 2014 - 2016 CatenaLogic. All rights reserved.
 // </copyright>
 // --------------------------------------------------------------------------------------------------------------------
 
-
 using Catel.Logging;
 using NUnit.Framework;
 
 [SetUpFixture]
 public class GlobalInitialization
 {
-    //[SetUp]
+    ////[SetUp]
```

Comment:
We certainly can disable the rule. IMO the rule doesn't get in the way of commenting code. I got used to double-commenting (which is just Ctrl+K,Ctrl+D with an extra strike to the D) years ago and it's occasionally useful later on when you're searching for commented code to have a dedicated way to do it (search for "////"). And I've never seen the tacky "//missing space" comments since.